node/node.go

```diff
@@ -130,6 +133,109 @@ func StateProvider(stateProvider statesync.StateProvider) Option {
 	}
 }
 
+// This function is expected to synchronize CometBFTs
+// stores with the application after statesync has been performed offline
+// It is expected that the blockstore and statestore are empty at
+// the time the function is called.
+// In case the blockstore is not empty, the function returns an error
+func BootstrapState(ctx context.Context, config *cfg.Config, dbProvider cfg.DBProvider, height uint64, appHash []byte) error {
```

Comment:
How do we anticipate this functionality will eventually make its way to operators so they can actually use it? Is the assumption that the SDK will make use of it? And if so, what about non-SDK users?

Reply:
In my view, if non-SDK users come to us asking for offline state sync we will need to prioritize the full Comet-based solution described as the main solution in [ADR-104](https://github.com/cometbft/cometbft/blob/main/docs/architecture/adr-104-out-of-band-state-sync.md).